admin_console/forms/PartnerConfiguration.php

```diff
@@ -591,6 +592,11 @@ public function init()
 			'decorators' => array('ViewHelper', array('Label', array('placement' => 'append')), array('HtmlTag',  array('tag' => 'dt', 'class' => 'partner_configuration_checkbox_field_only')))
 		));
 		
+		$this->addElement('checkbox', 'ALLOW_SSO_PER_USER', array(
```

Comment:
will prefer to have this in lowercase like the rest of the element names

Reply:
Because the permission name on admin.ini is all caps, then here it is all caps.
So it will be an exception either be here as caps, or on admin.ini as lowercase.